exaudfclient/BUILD

```diff
@@ -1,81 +1,108 @@
-load("@script_languages//:variables.bzl", "VM_ENABLED_DEFINES")
+load("//:variables.bzl", "VM_ENABLED_DEFINES")
 
-alias(
+config_setting(
     name = "benchmark",
-    actual = "@script_languages//:benchmark"
+    define_values = {"benchmark": "true"},
 )
 
-alias(
+config_setting(
     name = "python",
-    actual = "@script_languages//:python"
+    define_values = {"python": "true"},
 )
 
-alias(
+config_setting(
     name = "java",
-    actual = "@script_languages//:java"
+    define_values = {"java": "true"},
 )
 
-alias(
+config_setting(
     name = "bash",
-    actual = "@script_languages//:bash"
+    define_values = {"bash": "true"},
 )
 
 config_setting(
     name = "r",
     define_values = {"r": "true"},
 )
 
+config_setting(
+    name = "fast_binary",
+    define_values = {
+        "binary_type": "fast_binary",
+        },
+)
+
+config_setting(
+    name = "slow_wrapper",
+    define_values = {
+        "binary_type": "slow_wrapper",
+        },
+)
+
+
+config_setting(
+    name = "valgrind_wrapper",
+    define_values = {
+        "wrapper_type": "valgrind_wrapper",
+        },
+)
+
+config_setting(
+    name = "valgrind_massif_wrapper",
+    define_values = {
+        "wrapper_type": "valgrind_massif_wrapper",
+        },
+)
+
+config_setting(
+    name = "stdout_to_bucketfs",
+    define_values = {
+        "wrapper_type": "stdout_to_bucketfs",
+        },
+)
+
 
 VM_ENABLED_DEPS=select({
-        "//:benchmark": ["@script_languages//benchmark_container:benchmark_container"],
+        "//:benchmark": ["//base/benchmark_container:benchmark_container"],
         "//conditions:default": []
     }) + select({
-        "//:java": ["@script_languages//javacontainer:javacontainer"],
+        "//:java": ["//base/javacontainer:javacontainer"],
         "//conditions:default": []
     }) + select({
-        "//:bash": ["@script_languages//streaming_container:streamingcontainer"],
+        "//:bash": ["//base/streaming_container:streamingcontainer"],
         "//conditions:default": []
     })
 
-VM_PYTHON2_DEPS=select({
-        "//:python": ["@script_languages//python/python2:pythoncontainer"],
-        "//conditions:default": []
-    }) 
-
 VM_PYTHON3_DEPS=select({
-        "//:python": ["@script_languages//python/python3:pythoncontainer"],
+        "//:python": ["//base/python/python3:pythoncontainer"],
         "//conditions:default": []
-    }) 
+    })
+
 
 VM_R_DEPS=select({
         ":r": ["//rcontainer:rcontainer"],
         "//conditions:default": []
     })
 
-R_VM_ENABLED_DEFINE=select({
-        "//:r": ["ENABLE_R_VM"],
-        "//conditions:default": []
-    })
-VM_ENABLED_DEFINES=VM_ENABLED_DEFINES+R_VM_ENABLED_DEFINE
 
 cc_binary(
```

Comment:
Do we actually still need both exaudfclient_bin and exaudfclient_bin_py3

Reply:
I guess we don't. I will create a ticket for next Kehrwoche...

Reply:
See https://github.com/exasol/script-languages-release/issues/944